src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -214,6 +214,33 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 					timestamp: data.timestamp
 				},
 			};
+		case 'notification:show':
```

Comment:
it will generate 4 tables, maybe it is better to have one event `vscode_notification` with a property `action`

Reply:
done